### PR TITLE
Replace theme conditionals in templates with new skyIfTheme directive…

### PR DESCRIPTION
**Pull request by johnhwhite**
…, address affixer reference

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

File: src/app/public/modules/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -430,7 +430,7 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
         case SkyDropdownMessageType.Reposition:
           // Only reposition the dropdown if it is already open.
           /* istanbul ignore else */
-          if (this.isOpen) {
+          if (this.isOpen && this.affixer) {
```
Comment (johnhwhite) [inline src/app/public/modules/dropdown/dropdown.component.ts:433]: Fix for #137

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

File: src/app/public/modules/dropdown/dropdown.component.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/dropdown/dropdown.component.ts:433]: Can we write a unit test to check for the scenario we fixed?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**